File: CoreJavaAssignment/CoreJavaAssignment2/Car.java
package CoreJavaAssignment2;

public class Car {
		int speed;
		double regularPrice;
		String color;
		Car(int speed,double regularPrice,String color ){
			this.speed=speed;
			this.regularPrice=regularPrice;
			this.color=color;
		}
		public double getSalePrice()
		{
			return regularPrice;
		}
		public static void main(String args[])
		{
			Car c=new Car(300,2000000,"black");
			System.out.println(c.getSalePrice());
		}

	}

File: typescriptAssignment/Car.ts
export class Car{

    speed:number;
    regPrice:number;
    color:string;
    constructor(s:number,p:number,c:string)
    {
        this.speed=s;
        this.regPrice=p;
        this.color=c;
    }
   
    getSalePrice():number
    {
        return this.regPrice;
    }
    display()
    {
        console.log(this.speed);
        console.log(this.regPrice);
        console.log(this.color);
    }
}